dtls_debug.c

```diff
@@ -130,7 +130,7 @@ dtls_strnlen(const char *s, size_t maxlen) {
  */
 static size_t
 dsrv_print_addr(const session_t *addr, char *buf, size_t len) {
-#ifdef HAVE_INET_NTOP
+#if defined(HAVE_INET_NTOP) && !defined(WITH_LWIP)
```

Comment:
Just as general remark:
if the "auto-detection" of cmake/autoconf fails for some platforms, maybe placing the "os fixes" in "tinydtls.h" helps to remove that stuff from other code. (Not with this PR, but may be afterward.) 

What do others think about that?

Reply:
I did struggle with this one. 

My non-LwIP specific build environment did find inet_ntop and hence HAVE_INET_NTOP.  Looking to see if I can easily make the code cleaner.

Reply:
OK, so it's more caused by "cross-compiling".